### PR TITLE
Add dummy json quiz template

### PR DESCRIPTION
**Pull request by chrichri17**
Just for testing. I'll use this as a base template to implement the components

### CODE REVIEW

**Comment by chrichri17** [comment]
Current dependencies on/for this PR:
* main
  * **PR #30** <a href="https://app.graphite.dev/github/pr/YacineSteeve/quizoot/30" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/YacineSteeve/quizoot/30?utm_source=stack-comment).

File: frontend/src/data/quiz-data-types.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                "keywords": []
+            },
+            "grading": {
+                "point_value": 3,
+                "feedback": {
+                    "explanation": "'\\t' is the escape sequence for the Horizontal Tab character.\n\n'\\n' is the escape sequence for the Newline character.\n\nTo specify a character by hexadecimal value, use '\\xhh', where hh specifies the hexadecimal digits.\n\nHere are a few possible correct answers :\n- \"\\t\\n\\x7E\"\n- '\\t\\n\\x7E'\n- '''\\t\\n\\x7E'''"
+                }
+            },
+            "difficulty": "MEDIUM",
+            "tags": [],
```
Comment (YacineSteeve) [inline frontend/src/data/quiz-data-types.json]: Maybe you can add some tags for one question.
Here for example:
```
{
    ...
    "tags": ["string", "ascii"],
    ... 
}
```

**Review by YacineSteeve — APPROVED** [review]
That's a good idea

File: frontend/src/data/quiz-data-types.json (hunk 1 above)
Comment (chrichri17) [reply frontend/src/data/quiz-data-types.json]: I like that idea 💡

**Pull request merged**